src/app/market/sell/sell.component.ts

```diff
@@ -182,4 +182,9 @@ export class SellComponent implements OnInit {
     return this.pages[0].pageNumber;
   }
 
+  // Delete the listing using index
+  deleteListing(listIndex: number, index: number) {
```

Comment:
Use pageIndex, and listingIndex as the keywords
Also fix the order, page goes before listing